web/packages/shared/components/ClusterDropdown/ClusterDropdown.tsx

```diff
@@ -147,9 +163,13 @@ export function ClusterDropdown({
         </ButtonSecondary>
       </HoverTooltip>
       <Menu
-        popoverCss={() => `margin-top: 36px;`}
+        popoverCss={menuCss}
+        // the vertical here is ignored because it was being incorrectly calculated based on the menu
+        // height. It would cover the input and push above the screen, regardless of option. Instead, I
+        // explicitly overwrote the `top` styling in `menuCss`.
+        // TODO (avatus): fix popover top calculations
```

Comment:
just as an fyi when you come back to tackle this, it still gets pushed up when a error banner renders